### DISCUSSION TITLE
/xuexiluxian/os.html

### DISCUSSION THREAD

**Discussion by giscus** [itwanger/tobebetterjavaer-giscus#43] [Announcements]
# /xuexiluxian/os.html

操作系统是管理计算机硬件和软件资源的程序，是计算机的基石，这里整理一份 操作系统的超详细学习路线，附学习资源，可下载

https://tobebetterjavaer.com/xuexiluxian/os.html

**Comment by chanmufeng** [comment]
666

**Comment by wang15214086863** [comment]
嘤嘤嘤
